tests/Resources/ti.ui.view.test.js

```diff
@@ -1125,6 +1125,37 @@ describe('Titanium.UI.View', function () {
 			win.add(outerView);
 			win.open();
 		});
+
+		it.ios('1 value to create circle', finish => {
```

Comment:
why limit this to iOS? Presumably we want cross-platform consistency. I ran this locally on an emulator with 3.5 density and it appears to line up with iOS.

![borderRadius30px_30px_3 5x](https://user-images.githubusercontent.com/3252/91490161-ffe66000-e87f-11ea-9f10-facb6b695df9.png)

Reply:
Added.